ui/StatusQ/src/StatusQ/Components/StatusGroupBox.qml

```diff
@@ -0,0 +1,69 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+import utils 1.0
+
+
+GroupBox {
```

Comment:
It would be fine if you also add a description here!

Reply:
It's a standard GroupBox I'd say :)